libraries/EpoxyFS/src/EpoxyFS.h

```diff
@@ -195,7 +195,7 @@ class EpoxyDirImpl: public DirImpl {
 #ifdef _DIRENT_HAVE_D_TYPE
       return dirEntry_->d_type == DT_REG;
 #else
-      return (dirEntry_) ? S_ISREG(stat_.st_mode) : true;
+      return (dirEntry_) ? S_ISREG(stat_.st_mode) : false;
```

Comment:
It's been almost a year since I looked at this code, but I think code path for `(dirEntry_ == nullptr)` is not necessary. Because the client is expected to call `next()` before calling any of these other methods, and `next()` returns `false` if `dirEntry_ == nullptr`.